handbook/product/README.md

```diff
@@ -347,11 +348,16 @@ Requests are weighed by:
 - How well the request fits within Fleet's product vision and roadmap
 - Whether the feature seems like it can be designed, estimated, and developed in 6 weeks, given its individual complexity and when combined with other work already accepted
 
+### Customer Feature Requests 
```

Comment:
https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case

Reply:
Thanks! Changed to sentence case.